src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -190,4 +199,63 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
     const tnsr::ijj<DataType, SpatialDim, Frame>&
         deriv_spatial_metric) noexcept;
 
+namespace Tags {
+/*!
+ * \brief Compute item for spacetime metric \f$\psi_{ab}\f$ from the
+ * lapse \f$N\f$, shift \f$N^i\f$, and spatial metric \f$g_{ij}\f$..
```

Comment:
there is an extra period at the end of the line.

Reply:
removed